### PR TITLE
added text utility component

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently.

### CODE REVIEW

File: Text Utilities/textUtilities.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.grey-text {
+  font-size: 2.5rem;
+  font-weight: bold;
+  color: var(--btn-secondary-color);
+}
+
+.centered-text {
+  font-size: 4.5rem;
```
Comment (RakshithBhat03) [inline Text Utilities/textUtilities.css:56]: Don't add `font-size` when it comes to centering text.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a change.Everything else looks good

File: Text Utilities/textUtilities.css (hunk 1 above)
Comment (MonikaJhingan) [reply Text Utilities/textUtilities.css:56]: OK I will keep it in mind. Thanks for the suggestion.

File: Text Utilities/textUtilities.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  align-items: center;
```
Comment (ajay-mandaviya) [inline Text Utilities/textUtilities.css:57]: you can  used a t  text-align: center;  for center a text

**Review by ajay-mandaviya — APPROVED** [review]
everything looks good

**Review by aamrapaliwandhre — APPROVED** [review]

**Pull request closed without merging**